docs/model_server_rest_api.md

```diff
@@ -180,4 +183,103 @@ A request in [column format](https://www.tensorflow.org/tfx/serving/api_rest#spe
   "outputs": <value>|<(nested)list>|<object>
 }
 ```
-Read more about *Predict API* usage [here](./../example_client/README.md#predict-api-1)
\ No newline at end of file
+Read more about *Predict API* usage [here](./../example_client/README.md#predict-api-1)
+
+## Config Reload API <a name="config-reload"></a>
+* Description
+
+Sends requests via RESTful API to trigger config reloading and get models and DAGs statuses as a response.
+Flow after receiving request:
+1) If config file was changed - reload config. (If reload was needed return code is 201, otherwise 200)
+2) If any model version directory was changed or new version directory was added - reload this model.
+3) If any model that is part of a pipeline was changed or new version dir was added - reload this pipeline.
+4) Trigger Get Model Status API for all models and pipelines, aggregate all responses into one and return it with proper code.
+
+If any of above steps fail - error message with code 412 is returned.
+
+* URL
+```
+POST http://${REST_URL}:${REST_PORT}/v1/config/reload
+```
+* Request
+To trigger reload HTTP POST request should be sent on given URL.
+
+
+* Response
+In case of config reload success response contains aggregation of getModelStatus responses for all models and DAGs after reload is finished, along with operation status: 
```

Comment:
```suggestion
In case of config reload success, response contains aggregation of getModelStatus responses for all models and DAGs after reload is finished, along with operation status: 
```